modules/restoring-the-3scale-operator-secrets-and-apimanager.adoc

```diff
@@ -121,13 +136,21 @@ spec:
   - backuprepositories.velero.io
   itemOperationTimeout: 4h0m0s
 ----
+<1> Restoring the `Secret` backup.
```

Comment:
explanatory text, e.g., "Use the name you specified in the `metadata.name` field of the backup YAML", etc.